src/pycountry/__init__.py

```diff
@@ -3,10 +3,12 @@
 import os.path
 import unicodedata
 from importlib import metadata as _importlib_metadata
-from typing import Dict, List, Optional, Type
+from typing import cast, Dict, List, Optional, Type
 
 import pycountry.db
 
+from pycountry.db import Country as Country, Subdivision as Subdivision
```

Comment:
Allow these to be accessed from a base export like the other classes defined in this module